pom.xml

```diff
@@ -358,16 +362,19 @@
                   </bind>
                 </volumes>
                 <env>
-                  <!-- Test service IPs inside the containers use the Docker bridge IP address -->
-                  <AUTH_COOKIE_SERVICE>http://172.17.0.1:${test.hauth.port}/cookie</AUTH_COOKIE_SERVICE>
-                  <AUTH_TOKEN_SERVICE>http://172.17.0.1:${test.hauth.port}/token</AUTH_TOKEN_SERVICE>
-                  <!-- The Auth Access Service URI will have the ID to be checked appended onto the end -->
-                  <AUTH_ACCESS_SERVICE>http://172.17.0.1:${test.hauth.port}/access</AUTH_ACCESS_SERVICE>
+                  <!-- Cantaloupe configuration -->
                   <CANTALOUPE_LOG_APPLICATION_LEVEL>debug</CANTALOUPE_LOG_APPLICATION_LEVEL>
                   <CANTALOUPE_ENDPOINT_ADMIN_SECRET>secret</CANTALOUPE_ENDPOINT_ADMIN_SECRET>
                   <CANTALOUPE_ENDPOINT_ADMIN_ENABLED>true</CANTALOUPE_ENDPOINT_ADMIN_ENABLED>
                   <CANTALOUPE_DELEGATE_SCRIPT_ENABLED>true</CANTALOUPE_DELEGATE_SCRIPT_ENABLED>
                   <CANTALOUPE_DELEGATE_SCRIPT_PATHNAME>/usr/local/cantaloupe/delegate.jar</CANTALOUPE_DELEGATE_SCRIPT_PATHNAME>
+                  <!-- Delegate configuration -->
+                  <!-- Test service IPs inside the containers use the Docker bridge IP address -->
+                  <!-- The Auth Access Mode Service URI will have the ID to be checked appended onto the end -->
+                  <AUTH_ACCESS_MODE_SERVICE>http://172.17.0.1:${test.hauth.port}/access</AUTH_ACCESS_MODE_SERVICE>
```

Comment:
What's the value of adding MODE to this? I think it's already disambiguated from the other service endpoints.